t/html/pfappserver/cypress/support/commands.js

```diff
@@ -0,0 +1,73 @@
+Cypress.Commands.add('pfSystemLogin', () => {
+  /*
+  cy.visit('https://localhost:1443/admin#/login').then(() => {
+    cy.readFile('/usr/local/pf/conf/unified_api_system_pass').then((password) => {
+      cy.get('form input#username').first().type('system')
+      cy.get('form input#password').first().type(password)
+      cy.get('form button[type="submit"]').first().click()
+    })
+  })
+  */
+  return cy.pfUnifiedSystemPassword().then(password => {
+    return cy.request('POST', '/api/v1/login', { username: 'system', password }).then(response => {
+      return window.localStorage.setItem('user-token', response.body.token)
+    })
+  })
+})
```

Comment:
Are we using API or the browser?

Reply:
The configurator is using the Browser, which doesn't require /login

This was created during the PoC to provide a support helper for writing tests. I left it there to help with future development.